sumpy/kernel.py

```diff
@@ -933,6 +933,49 @@ def get_pde_as_diff_op(self):
         return laplacian(w)
 
 
+class GaussTransformKernel(ExpressionKernel):
+    init_arg_names = ("dim",)
+
+    def __init__(self, dim, gauss_delta_name="delta"):
```

Comment:
Could you add a P2P test to ensure it satisfies the PDE? (Given that $\delta$ is a parameter, you'll probably need multiple rounds of P2P.)